src/katex.less

```diff
@@ -401,8 +401,13 @@
             text-align: center;
         }
 
-        .accent-body {
+        // Accents that are not of the accent-full class have zero width
+        // (do not contribute to the width of the final symbol).
+        .accent-body:not(.accent-full) {
             width: 0;
+        }
```

Comment:
I wish we store more props directly on the `domTree` nodes we create so that we don't have all of this extra data hanging out in this file.  No action required.